src/index.html

```diff
@@ -19,0 +1,15 @@
+<!DOCTYPE html>
+<div class="product-card" data-qa="card">
+  <div class="product-card__image product-card__image--product1"></div>
+  <div class="product-card__details">
+    <h2 class="product-card__title">Product Title</h2>
+    <div class="product-card__stars">
+      <!-- Stars block here -->
+    </div>
+    <p class="product-card__price">$19.99</p>
+    <a href="#"
+       class="product-card__buy"
+       data-qa="hover">BUY</a>
+  </div>
+</div>
+<link rel="stylesheet" href="styles.css">
```

Comment:
The stylesheet link should point to the correct SCSS file if SCSS is being used. Ensure that the SCSS files are correctly integrated.